src/utils/localizeDateHelper.ts

```diff
@@ -1,21 +1,34 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const tz = require('timezone')(require('timezone/zones'), require('timezone/en_US.js'), require('timezone/sv_SE.js'), require('timezone/fr_FR.js'), require('timezone/nb_NO.js'), require('timezone/de_DE'), require('timezone/es_ES'), require('timezone/ja_JP'), require('timezone/ko_KR'));
-
+/* eslint-disable @typescript-eslint/no-var-requires,global-require */
 // tz docs https://bigeasy.github.io/timezone/
+const tz = require('timezone')(
```

Comment:
This.  This is awesome.  Wondering if this pattern should be included in client docs for custom components for like best practices.